dvc/ui/__init__.py

```diff
@@ -75,16 +72,22 @@ def write(
         style: str = None,
         sep: str = None,
         end: str = None,
-        file: TextIO = None,
-        flush: bool = False,
+        stderr: bool = False,
         force: bool = False,
+        styled: bool = False,
     ) -> None:
+        sep = " " if sep is None else sep
+        end = "\n" if end is None else end
         if not self._enabled and not force:
             return
 
-        file = file or sys.stdout
+        if styled:
+            console = self.error_console if stderr else self.rich_console
+            return console.print(*objects, sep=sep, end=end)
```

Comment:
Note that `rich_console` expands the tab, whereas `print` does not. But, `logger.info` did, so we are still compatible.